rfml.go

```diff
@@ -476,6 +464,120 @@ func uploadRFMLFilesInDirectory(rfmlDirectory string) error {
 	return nil
 }
 
+func downloadRFML(c cliContext) error {
+	testDirectory := c.String("test-folder")
+	absTestDirectory, err := prepareTestDirectory(testDirectory)
+	if err != nil {
+		return cli.NewExitError(err.Error(), 1)
+	}
+
+	var mappings rainforest.TestIDMappings
+	mappings, err = api.GetRFMLIDs()
+	if err != nil {
+		return cli.NewExitError(err.Error(), 1)
+	}
+
+	var testIDs []int
+	if len(c.Args()) > 0 {
+		var testID int
+		for _, arg := range c.Args() {
+			testID, err = strconv.Atoi(arg)
+			if err != nil {
+				return cli.NewExitError(err.Error(), 1)
+			}
+
+			testIDs = append(testIDs, testID)
+		}
+	} else {
+		for _, testIDMap := range mappings {
+			testID := testIDMap.ID
+			testIDs = append(testIDs, testID)
+		}
+	}
+
+	errorsChan := make(chan error)
+	testIDChan := make(chan int, len(testIDs))
+	testChan := make(chan *rainforest.RFTest, len(testIDs))
+
+	for _, testID := range testIDs {
+		testIDChan <- testID
+	}
+	close(testIDChan)
+
+	for i := 0; i < rfmlDownloadConcurrency; i++ {
+		go downloadRFTestWorker(testIDChan, errorsChan, testChan)
+	}
+
+	for i := 0; i < len(testIDs); i++ {
+		select {
+		case err = <-errorsChan:
+			return err
+		case test := <-testChan:
+			test.UnmapBrowsers()
```

Comment:
Wouldnt it be better to have a single function to do all the necessary conversion, like `test.Init(mappings)`  rather then separate calls to `UnmapBrowsers`/`UmarshalElements`, as more need to be added later on @macocha @epaulet